.gitignore

```diff
@@ -1,10 +1,9 @@
-.DS_Store
-*.iml
-
 local.properties
 .gradle
-build
+build/
 
 .idea/
-!/.idea/codeStyles/*
```

Comment:
These don't actually do anything since the idea folder is excluded. Instead the files we want are explicitly tracked